nanoc/lib/nanoc/filters/sass/importer.rb

```diff
@@ -36,13 +36,23 @@ def find(identifier, options)
       items = filter.items.find_all(identifier)
       return if items.empty?
 
-      content = if items.size == 1
-                  items.first.compiled_content
-                else
-                  items.map { |item| %(@import "#{item.identifier}";) }.join("\n")
-                end
+      syntax, content = if items.size == 1
+                          item = items.first
+
+                          [
+                            item.identifier.ext.downcase.to_sym || :scss,
```

Comment:
I think the `|| :scss` case is not tested, and likely not correct either (if there is no extension, `.downcase` would raise a NoMethodError I expect).

Reply:
Good catch! In the extensionless case Nanoc now specifies `syntax = nil` which makes Sass use its internal default (which is Sass).